CHANGELOG.md

```diff
@@ -11,3 +11,9 @@ frontend TypeScript client, `django-ai-assistant-client`.
 ## 0.0.1 <small>June 25, 2024</small> {id="0.0.1"}
 
 - Initial release
+
+## 0.0.2 <small>June 28, 2024</small> {id="0.0.2"}
+
+- Add support for Django 4.2 LTS
+- Add support for Python 3.10 and 3.11
```

Comment:
Should we mention the PRs or the changes made to the examples?

Reply:
Can you change the order? Newer versions on top.

Reply:
> Should we mention the PRs or the changes made to the examples?

No need to.